robin_stringer/server.js

```diff
@@ -0,0 +1,14 @@
+
+var mongoose = require('mongoose');
+var express = require('express');
+var app = express();
+var authorRouter = require(__dirname + '/routes/authorroutes');
+var bookRouter = require(__dirname + '/routes/bookroutes');
+
+mongoose.connect(process.env.MONGOLAB_URI || 'mongodb://localhost/book_dev');
+
```

Comment:
Now that you have a /build folder, you need to serve up the html and bundle.js that is in the folder (as static content). You do it right about here. :)